### PR TITLE
doc: Add common CA certs to reference

### LINKED ISSUE #3163: Add (up-to-date) Root CA PEM file to Documentation
#### Summary

We should add a PEM file to our documentation site containing the Root CAs for typical deployments.

#### Why do we need this ?

- Basic Station Gateways require a "trust" file instead of relying on the roots in the OS.
- Some MQTT clients require a "CA" file instead of relying on the roots in the OS.
- For many users it's too difficult to find out which CA(s) they need to include for a given deployment.

#### What is already there? What do you see now?

We currently link to the certificate of `Let's Encrypt Authority X3` signed by `DST Root CA X3` which expires `Mar 17 2021`. This would mean that all these gateways would have to be updated with a new root before then.

#### What is missing? What do you want to see?

I think we should put a `ca.pem` file on our documentation site that contains the most used CAs in typical deployments (i.e. following the documented deployment methods).

#### How do you propose to document this?

- Determine a set of Root CAs to support. I'd propose `DST Root CA X3` (which we already link to), `ISRG Root X1` (for future Let's Encrypt), and the roots used to sign certificates on the major cloud providers.
	- Alternatively, we could take Mozilla's common root CAs, but that file might be too big for some gateways.
- Create a `.pem` file with those certificates.
- Link to it in the docs.

#### Can you do this yourself and submit a Pull Request?

I think this is something for @benolayinka or @nejraselimovic.

Adding @johanstokking @KrishnaIyer @adriansmares @rvolosatovs for discussion

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes #3163 

#### Testing

Run locally

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/root-certificates/ca.pem [hunk 1]
```diff
@@ -0,0 +1,146 @@
+DST Root CA X3
+==============
```
Comment (KrishnaIyer) [inline doc/content/reference/root-certificates/ca.pem:2]: This is not a valid `.pem` file. You should remove those headings and also the extra new line. Here's a [reference](https://github.com/ARMmbed/mbedtls/blob/development/tests/data_files/test-ca_cat21.crt)

If you want to tag which certificate is which, I'd recommend adding an extra `md` file somewhere and listing these in order.

**Review by KrishnaIyer — CHANGES_REQUESTED** [review]

File: doc/content/reference/root-certificates/ca.pem (hunk 1 above)
Comment (benolayinka) [reply doc/content/reference/root-certificates/ca.pem:2]: I think anything outside the certificate tags is ignored https://stackoverflow.com/questions/19578812/comments-in-a-pem-file

Comment (KrishnaIyer) [reply doc/content/reference/root-certificates/ca.pem:2]: That's not standard behaviour. Some clients (like `OpenSSL`) do that but not all.

Comment (htdvisser) [reply doc/content/reference/root-certificates/ca.pem:2]: Anything outside the boundaries is indeed ignored: https://tools.ietf.org/html/rfc7468#section-2

Comment (htdvisser) [reply doc/content/reference/root-certificates/ca.pem:2]: This is fine: https://tools.ietf.org/html/rfc7468#section-5.2

**Review by KrishnaIyer — APPROVED** [review]

**Pull request merged**